.github/workflows/java-pgjdbc-integ-tests.yml

```diff
@@ -1,4 +1,4 @@
-name: pgJDBC Integration tests
+name: Java pgJDBC Integration tests
```

Comment:
Nit - lowercase `integration` vs `Integration`, inconsistent in several of the workflows